src/main/java/io/personium/client/ServiceCollection.java

```diff
@@ -67,6 +137,58 @@ public ServiceCollection(Accessor as, String path) {
         super(as, path);
     }
 
+    /**
+     * This method generate XML string for configuration of service.
+     * @param routes list of route
+     * @param subject Value of the service subject
+     * @return String XML for configuration
+     */
+    public String createConfigureXML(List<IPersoniumServiceRoute> routes, String subject) {
+        StringWriter sw = new StringWriter();
+
+        try {
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+            factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+            factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "");
+            DocumentBuilder builder = factory.newDocumentBuilder();
+            DOMImplementation domImpl = builder.getDOMImplementation();
+
+            String nsD = "DAV:";
+            String nsP = "urn:x-personium:xmlns";
+
+            Document document = domImpl.createDocument(nsD, "D:propertyupdate", null);
```

Comment:
Is this method creates `<D:propertyupdate xmlns:D="DAV:" xmlns:p="urn:x-personium:xmlns">`?